mmpose/models/pose_estimators/base.py

```diff
@@ -17,13 +18,19 @@ class BasePoseEstimator(BaseModel, metaclass=ABCMeta):
             config of :class:`BaseDataPreprocessor`. Defaults to ``None``.
         init_cfg (dict | ConfigDict): The model initialization config.
             Defaults to ``None``
+        metainfo (dict): Meta information for dataset, such as keypoints
```

Comment:
May need further descriptions, like the metainfo from the input data batch will be ignored if this argument is set. And it's better to provide a link to the document of the dataset metainfo here.